### ISSUE TITLE
BT Mode searching for AD (audio device)

### ISSUE THREAD

**Issue by sakthia01** [AuthorizeNet/inperson-sdk-ios#88]
Set Connection mode to BT (Bluetooth) in settings bundle,  try to do an EMV transaction it is searching for an audio device and after some time it is throwing time out error with no audio device is found.
Why SDK is searching for audio device when I connected with bluetooth device.

**Comment by gramsejrm** [comment]
I'm having the same issue. Did you find a solution?

**Comment by sakthia01** [comment]
> I'm having the same issue. Did you find a solution?

Yes.  Change your default mode from AD to BT under settings. Goto your settings -> <YOUR APP> -> <Check for your default mode to BT instead of AD>.

Check the same in your code.
